app/src/main/java/com/hieuminh/chatapp/presenter/impl/ChatDetailPresenterImpl.kt

```diff
@@ -34,13 +34,17 @@ class ChatDetailPresenterImpl(private var view: ChatDetailContract.View?) :
     }
 
     override fun sendMessage(word: WordModel, chatId: String) {
-        messageModel?.messages?.add(0, word)
         val messagesMap = HashMap<String, Any>()
-        messageModel?.messages?.let { messagesMap.put(MESSAGES, it) }
+        if(words == null) {
+            messagesMap[MESSAGES] = listOf(word)
+        } else {
+            words?.add(0, word)
+            messagesMap[MESSAGES] = words!!
+         }
```

Comment:
avoid force wrap => !!
let get a trick like that: 
```suggestion
val words = mWords
        if(words == null) {
            messagesMap[MESSAGES] = listOf(word)
        } else {
            words.add(0, word)
            messagesMap[MESSAGES] = words
         }
```